src/stylesheets/application.scss

```
@@ -2,4 +2,14 @@
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
 
+@import 'variables';
+
+// base
+@import 'base/form';
+
+//layouts
+@import 'layouts/auth';
+@import 'layouts/default';
+
 @import 'screens';
+@import 'components';
```

Components must be imported before screens so that they can be overridden on a per-screen basis:

```suggestion
@import 'base';

@import 'layouts/auth';
@import 'layouts/default';

@import 'components';

@import 'screens';
```